src/runtime/components/forms/Form.vue

```diff
@@ -207,95 +207,100 @@ function isSuperStructSchema(schema: any): schema is Struct<any, any> {
   )
 }
 
-async function getYupErrors(
```

Comment:
Can we keep these functions? I think we can rename them and change their return values to `{ errors, state }` to return both the transformed state and the list of errors. We can rename them to `validateWithX` for example.

Reply:
okee thank you for feedback. i refactor it soon